src/namedlevels.jl

```diff
@@ -23,7 +23,7 @@ function lvls(indexer, names::Vararg{Symbol})
     name_set = first.(levelname_map(indexer))
     num_set = last.(levelname_map(indexer))
     map(names) do name
-        ind = findfirst(name_set, name)
+        ind = something(findfirst(isequal(name), name_set), 0)
```

Comment:
```suggestion
        ind = findfirst(isequal(name), name_set)
```

If this is not found then it is fine for the `ind` to be `nothing`, that will give a surficent error